components/feature/tab-collections/src/main/java/mozilla/components/feature/tab/collections/adapter/TabAdapter.kt

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.tab.collections.adapter
+
+import android.content.Context
+import mozilla.components.browser.session.SessionManager
+import mozilla.components.browser.session.ext.readSnapshotItem
+import mozilla.components.concept.engine.Engine
+import mozilla.components.feature.tab.collections.Tab
+import mozilla.components.feature.tab.collections.db.TabEntity
+
+internal class TabAdapter(
+    val actual: TabEntity
```

Comment:
Could renaming  `actual` to `tabEntity`  convey is purpose better? In the diff the first time that I saw `actual` been use from another class I thought that you  multiple instance and `actual` was the `actual` one.

Reply:
I renamed the property in both adapters to "entity". :)

Reply:
👍